src/components/progress-bar/__tests__/ProgressBar.test.tsx

```diff
@@ -1,3 +1,4 @@
+import { shallow } from 'enzyme';
 import React from 'react';
```

Comment:
We should have `import React from 'react';` as the first import.

Reply:
we should have prettier to do that for use in the first place...

Reply:
fixed

Reply:
Made to ticket to config prettier to do this